java/jre_ver.bzl

```diff
@@ -0,0 +1,9 @@
+def jre_ver(version):
+  """Extract JRE version from an OpenJDK debian package.
+    debian packages have versions in the form:
+       openjdk-8-jre*: 8u171-b11-1~deb9u1
+       openjdk-11-jre*: 11.0.1+13-2~bpo9+1
+  """
+  if version.startswith("8u"): return version.split("-",2)[0]
+  if version.startswith("11."): return version.split("+",2)[0]
+  return "unknown: " + version
```

Comment:
Hmm... would it be difficult to not set `JAVA_VERSION` if unknown?

Reply:
I'd actually like to trigger a build error.  Thinking to the day that we pull in Java 15 and end up with a broken version string.

Reply:
Ok, that should use the `fail()` function.

Reply:
Nice! I like that idea.

Reply:
`fail()` does the trick:
```
ERROR: /Users/bsd/Manumitting/Projects/GooglePluginEclipse/repo-distroless/java/BUILD:13:1: Traceback (most recent call last):
	File "/Users/bsd/Manumitting/Projects/GooglePluginEclipse/repo-distroless/java/BUILD", line 13
		container_image(name = rule_name, base = "//cc" if...", <5 more arguments>)
	File "/Users/bsd/Manumitting/Projects/GooglePluginEclipse/repo-distroless/java/BUILD", line 27, in container_image
		jre_ver(versions[jre_deb])
	File "/Users/bsd/Manumitting/Projects/GooglePluginEclipse/repo-distroless/java/jre_ver.bzl", line 10, in jre_ver
		fail(("unrecognized openjdk package v...))
unrecognized openjdk package version: 15.0.2
ERROR: error loading package 'java': Package 'java' contains errors
INFO: Elapsed time: 0.093s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
```